lib/voorhees/json_api.ex

```diff
@@ -50,4 +77,21 @@ defmodule Voorhees.JSONApi do
 
     actual
   end
+
+  defp _stringify_items([]), do: []
```

Comment:
:+1: perhaps pull into a Utility module for importing?

After review I would suggest merging this then I can do a refactor into a Utility module